packages/calling-component-bindings/src/callControlSelectors.ts

```diff
@@ -140,6 +141,38 @@ export type ScreenShareButtonSelector = (
   /* @conditional-compile-remove(capabilities) */ /* @conditional-compile-remove(PSTN-calls) */ disabled?: boolean;
 };
 
+/* @conditional-compile-remove(raise-hand) */
+/**
+ * Selector type for {@link RaiseHandButton} component.
+ *
+ * @public
+ */
+export type RaiseHandButtonSelector = (
+  state: CallClientState,
+  props: CallingBaseSelectorProps
+) => {
+  checked?: boolean;
+  /* @conditional-compile-remove(PSTN-calls) */
+  disabled?: boolean;
+};
+
+/* @conditional-compile-remove(raise-hand) */
+/**
+ * Selector for {@link RaiseHandButton} component.
+ *
+ * @public
+ */
+export const raiseHandButtonSelector: RaiseHandButtonSelector = reselect.createSelector(
+  [getLocalParticipantRaisedHand, /* @conditional-compile-remove(PSTN-calls) */ getCallState],
+  (raisedHand, /* @conditional-compile-remove(PSTN-calls) */ callState) => {
+    return {
+      checked: raisedHand ? true : false,
+      /* @conditional-compile-remove(PSTN-calls) */
+      disabled: callState === 'InLobby' ? true : callState === 'Connecting' ?? false
```

Comment:
potentially onhold as well?

Reply:
I agree this should probably be disabled in this state